### PR TITLE
Update

### PR DESCRIPTION
**Pull request by curtis-rainforestqa**
Updates a few rules

### CODE REVIEW

**Comment by curtis-rainforestqa** [comment]
@shosti sure thing, there isn't a dependency defined currently but I can add it.

**Comment by shosti** [comment]
Yeah just realized that, seems like a good idea though

**Comment by curtis-rainforestqa** [comment]
@shosti added

File: ruby/rubocop_lint.yml [hunk 1]
```diff
@@ -111,16 +111,6 @@ Lint/IneffectiveAccessModifier:
                  the visibility of a class method, which does not work.
   Enabled: true
 
-Lint/InvalidCharacterLiteral:
```
Comment (shosti) [inline ruby/rubocop_lint.yml]: what happened to these two?

Comment (curtis-rainforestqa) [reply ruby/rubocop_lint.yml]: looks like it wasn't really working...

https://github.com/bbatsov/rubocop/pull/4086/commits

**Pull request merged**